OasisDEX/oasis-borrow
# AAVE Position on owner page

# [AAVE Position on onwer page](https://app.shortcut.com/oazo-apps/story/5959/show-aave-steth-position-owner-page-earn-section)
  
## Changes 👷‍♀️
- Aave Positon is displayed within earn positions
  
## How to test 🧪
- Open Position
- Go to the owner page
- The header vaults indicator should increment.
